app/controllers/clinicfinders_controller.rb

```diff
@@ -4,10 +4,15 @@ class ClinicfindersController < ApplicationController
   def index; end
 
   def search
-    @abortron = Abortron::ClinicFinder.new("#{Rails.root}/clinics.yml")
-    @nearest = @abortron.locate_nearest_clinic patient_zipcode: params[:zip].to_i.to_s,
-                                               gestational_age: params[:gestation].to_i
-    @cheapest = @abortron.locate_cheapest_clinic(gestational_age: params[:gestation].to_i)
+    puts 'searching fasdlfndsl'
+    puts params
+    @abortron = ClinicFinder::Locator.new(Clinic.all,
```

Comment:
There are a few API changes here:

* Now takes `Clinic` objects
* the object is `ClinicFinder::Locator` instead of `Abortron` ( 😢  )
* Some changes in the args for everything

Reply:
Also need to pass in API keys. Can't remember if I do em by env var in the gem or not.